src/PactNet.Native/NativePactBuilder.cs

```diff
@@ -141,7 +141,8 @@ private Uri StartMockServer()
             // TODO: add TLS support
             int serverPort = this.server.CreateMockServerForPact(this.pact, address, false);
 
-            var uri = new Uri($"http://{this.host}:{serverPort}");
+            var mockServerUrl = $"http://{hostIp}:{serverPort}";
+            var uri = new Uri(mockServerUrl);
```

Comment:
Minor, but any reason to not just inline the return i.e. `return new Uri(...)`?

Reply:
Because I never would've tracked down the problem otherwise 😆 Having those are separate vars allows you to eyeball them with the debugger, but no perf hit or anything in a Release build.